lib/rules/font-family-name-quotes/index.js

```diff
@@ -56,8 +56,67 @@ function quotesRequired(family) {
 		.some((word) => /^(?:-?\d|--)/.test(word) || !/^[-\w\u{00A0}-\u{10FFFF}]+$/u.test(word));
 }
 
+/**
+ * @typedef {{
+ * name: string,
+ * hasQuotes: boolean,
+ * sourceIndex: number,
+ * removeQuotes: function,
+ * addQuotes: function
```

Comment:
[suggest] How about the following refactoring ideas?

- Make indentation for each property
- Avoid `function` for more clarity

```suggestion
 *   name: string,
 *   hasQuotes: boolean,
 *   sourceIndex: number,
 *   removeQuotes: () => void,
 *   addQuotes: () => void,
```